src/deno_dir.rs

```diff
@@ -165,12 +171,11 @@ impl DenoDir {
 
```

Comment:
These lines above are meant to support http imports - they correspond to these two lines in the prototype
https://github.com/denoland/deno/blob/559453cf6cc88283bcf8fdeccd387458f5c63165/os.go#L76-L77
(I haven't yet grokked how you're handling this - but generally I'm trying to follow the prototype so that we can copy tests from there.)